core/osrd-railjson/src/main/java/fr/sncf/osrd/railjson/schema/infra/trackranges/RJSRange.java

```diff
@@ -0,0 +1,28 @@
+package fr.sncf.osrd.railjson.schema.infra.trackranges;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import fr.sncf.osrd.railjson.schema.common.graph.ApplicableDirection;
+import java.util.Comparator;
+import java.util.List;
+
+@SuppressFBWarnings({"URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD"})
+public class RJSRange {
+    /** Positions from the beginning of the RJSTrackSection */
+    public double begin;
+    public double end;
+
+    public RJSRange(double begin, double end) {
+        this.begin = begin;
+        this.end = end;
+    }
+
+    /** Check whether a list of track ranges is overlaping */
+    public static boolean isOverlaping(List<? extends RJSRange> ranges) {
```

Comment:
Typo in Overlapping

Not sure about the rules for documentation in OSRD, is there a doc ? Otherwise I think it's cool to have very explicit names and remove paraphrasing docs
```suggestion
    public static boolean areRangesOverlapping(List<? extends RJSRange> ranges) {
```

Reply:
I don't think this is used, it is heritage, I did not delete it, maybe I should

Reply:
@Castavo you can ditch it